services/movie_session.py

```diff
@@ -0,0 +1,53 @@
+from db.models import MovieSession, Movie, CinemaHall
+
+from django.db.models import QuerySet
+
+from datetime import datetime
+
+
+def create_movie_session(
+        movie_show_time: datetime,
+        movie_id: int,
+        cinema_hall_id: int
+) -> MovieSession:
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie=Movie.objects.get(id=movie_id),
+        cinema_hall=CinemaHall.objects.get(id=cinema_hall_id)
+    )
+
+
+def get_movies_sessions(
+        session_date: datetime = None
+) -> QuerySet[MovieSession] | MovieSession:
```

Comment:
The return type hint `QuerySet[MovieSession] | MovieSession` is incorrect. Since `get_movies_sessions` always returns a QuerySet, it should be `QuerySet[MovieSession]`.